src/commands/economy/GiveCommand.js

```diff
@@ -13,44 +13,39 @@ module.exports = class GiveCommand extends Command {
   async giveStar(user, user2, valor, ctx, mencionado) {
     if (valor > user.estrelinhas) return ctx.reply('error', `${ctx.locale('commands:give.poor')} ${ctx.locale('commands:give.stars')}`);
 
-    await this.client.database.Users.updateOne({ id: user.id }, { $inc: { estrelinhas: -valor } });
-    await this.client.database.Users.updateOne({ id: user2.id }, { $inc: { estrelinhas: valor } });
+    await this.client.repositories.giveRepository.giveStars(user.id, user2.id, valor);
 
     ctx.reply('success', `${ctx.locale('commands:give.transfered', { value: valor, emoji: '⭐' })} ${mencionado}`);
   }
 
   async giveDemon(user, user2, valor, ctx, mencionado) {
     if (valor > user.caçados) return ctx.reply('error', `${ctx.locale('commands:give.poor')} ${ctx.locale('commands:give.demons')}`);
 
-    await this.client.repositories.userRepository.update(user.id, { $inc: { caçados: -valor } });
-    await this.client.repositories.userRepository.update(user2.id, { $inc: { caçados: valor } });
+    await this.client.repositories.giveRepository.giveDemons(user.id, user2.id, valor);
 
     ctx.reply('success', `${ctx.locale('commands:give.transfered', { value: valor, emoji: '<:Demon:758765044443381780>' })} ${mencionado}`);
   }
 
   async giveAngel(user, user2, valor, ctx, mencionado) {
     if (valor > user.anjos) return ctx.reply('error', `${ctx.locale('commands:give.poor')} ${ctx.locale('commands:give.angels')}`);
 
-    await this.client.repositories.userRepository.update(user.id, { $inc: { anjos: -valor } });
-    await this.client.repositories.userRepository.update(user2.id, { $inc: { anjos: valor } });
+    await this.client.repositories.giveRepository.giveAngels(user.id, user2.id, valor);
 
     ctx.reply('success', `${ctx.locale('commands:give.transfered', { value: valor, emoji: '<:Angel:758765044204437535>' })} ${mencionado}`);
   }
 
   async giveSD(user, user2, valor, ctx, mencionado) {
     if (valor > user.semideuses) return ctx.reply('error', `${ctx.locale('commands:give.poor')} ${ctx.locale('commands:give.semigods')}`);
 
-    await this.client.repositories.userRepository.update(user.id, { $inc: { semideuses: -valor } });
-    await this.client.repositories.userRepository.update(user2.id, { $inc: { semideuses: valor } });
+    await this.client.repositories.giveRepository.giveSemigods(user.id, user2.id, valor);
```

Comment:
`Semideuses` in english is `demigods`, so if we are going to make the code english-only, lets use all variables in english
```suggestion
    await this.client.repositories.giveRepository.giveDemigods(user.id, user2.id, valor);
```